tests/testsuite/doc.rs

```diff
@@ -1464,3 +1464,33 @@ fn doc_workspace_open_help_message() {
             .with_stderr_contains("  bar"),
     );
 }
+
+#[test]
+fn doc_edition() {
+    if !cargotest::is_nightly() {
+        // Stable rustdoc won't have the edition option.  Remove this once it
+        // is stabilized.
+        return;
+    }
+    let p = project("foo")
+        .file(
+            "Cargo.toml",
+            r#"
+            cargo-features = ["edition"]
+            [package]
+            name = "foo"
+            version = "0.0.1"
+            authors = []
+            rust = "2018"
+        "#,
+        )
+        .file("src/lib.rs", "")
+        .build();
+
+    assert_that(
+        p.cargo("doc").arg("-v").masquerade_as_nightly_cargo(),
```

Comment:
You can write this compactly as `p.cargo("doc -v").masquerade_as_nightly_cargo()`, `cargo` function does simple argument splitting.